guides/ruby_on_rails_quick_start.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Ruby on Rails quick start — Meilisearch documentation
+description: Integrate Meilisearch into your Ruby on Rails app.
+---
+
+# Ruby on Rails quick start
+
+Integrate Meilisearch into your Ruby on Rails app.
+
+## 1. Create a Meilisearch project
+
+[Create a project](https://cloud.meilisearch.com) in the Meilisearch Dashboard. Check out our [getting started guide](/learn/getting_started/cloud_quick_start) for step-by-step instructions.
+
+## 2. Create a Rails app
+
+Ensure your environment uses at least Ruby 2.7.0 and Rails 6.1.
+
+```bash
+rails new blog
+```
+
+## 3. Install the meilisearch-rails gem
+
+Navigate to your Rails app and install the `meilisearch-rails` gem.
+
+```bash
+bundle add meilisearch-rails
+```
+
+## 4. Add your Meilisearch credentials
+
+Run the following command to create a `config/initializers/meilisearch.rb` file.
+
+```bash
+bin/rails meilisearch:install
+```
+
+Then add your Meilisearch URL and [Default Admin API Key](/learn/security/basic_security#obtaining-api-keys). On Meilisearch Cloud, you can find your credentials in your project settings.
+
+```Ruby
+MeiliSearch::Rails.configuration = {
+  meilisearch_url: '<your Meilisearch URL>',
+  meilisearch_api_key: '<your Meilisearch API key>'
+}
+```
+
+## 5. Generate the model and run the database migration
+
+Create an example Article model and generate the migration files.
```

Comment:
Seems like we use backticks when mentioning the model elsewhere in this page, so better to be consistent.

```suggestion
Create an example `Article` model and generate the migration files.
```